src/container.ts

```diff
@@ -16,7 +17,7 @@ import { last, nameValueToObject, isClass } from './utils'
 import { InjectionMode, InjectionModeType } from './injection-mode'
 import { Lifetime } from './lifetime'
 import { AwilixResolutionError, AwilixTypeError } from './errors'
-
+import { importModule } from './load-module-native.js'
```

Comment:
Tiny nit: newline for whitespace between imports and impl. Did you run `npm run lint` btw?

Reply:
I did, it doesn't give an error for this :(

Reply:
I think it also runs prettier but I guess it didn’t fix this

Reply:
Yep, I fixed it manually though